metadata/src/main/java/com/redhat/lightblue/metadata/FieldConstraint.java

```diff
@@ -31,4 +31,10 @@
      * Determines if the constraint is valid for the given field type
      */
     boolean isValidForFieldType(Type fieldType);
+
+
+    /**
+     * Returns the description of the FieldConstraint (for documentation propose)
+     */
+    String describeConstraint();
```

Comment:
I will change it. I used 'to describe' (the verb) intended to say that could be dynamic generated (using interpolation of different things to compose a meaning description) .